Sources/WishKit/SwiftUI/CreateWishView.swift

```diff
@@ -192,7 +192,7 @@ struct CreateWishView: View {
         }
         .frame(maxWidth: .infinity, maxHeight: .infinity)
         .background(backgroundColor)
-        .ignoresSafeArea(edges: [.leading, .bottom, .trailing])
+        .ignoresSafeArea(edges: [.leading, .trailing])
```

Comment:
There might have been a rationale behind this. I think we're more on the safe side if you define it for specific regions. What happens if you do `.ignoresSafeArea(.container, edges: [.leading, .bottom, .trailing])`?

Reply:
Right. Maybe you can tell the inner scrollview to respect the safe area this way. There is also the `.keyboard` region, maybe that helps.